canonical/ubuntu.com
# Add plausible and custom event to track

## Done

- Add plausible and custom events on each step of the product selector page.

## QA

- Go to https://plausible.io/ubuntu.com 
- Go to /pro/subscribe, select items and click buy button
- Check the dashboard number changes appropriately with selected items when buy button is clicked.
- Also check other pages are not tracked, other than /pro, /pro/dashboard, pro/subscribe 
   (It might be better to QA after removing all the data on plausible dashboard)
- Refer to this documentation if needed - https://plausible.io/docs

## Issue / Card

Fixes #https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/801?modal=detail&selectedIssue=WD-3161
